UPGRADE-7.0.md

```diff
@@ -0,0 +1,207 @@
+UPGRADE FROM 6.3 TO 7.0
+=======================
+
+# Removed Code Entities
+
+The code entities in this list were marked as deprecated in previous releases and have now been removed.
+
+## Services
+
+## Container Parameters
+
+## Bundle Configuration
+
+## Constants
+
+- CHAMELEON_MEMCACHE_USE_LAZY_CACHE_WRITE_WRITEQUEUE
+- CHAMELEON_MEMCACHE_USE_LAZY_CACHE_WRITE_WRITEQUEUE_DIR
+- chameleon::REQUEST_TYPE_ASSETS
+- chameleon::REQUEST_TYPE_BACKEND
+- chameleon::REQUEST_TYPE_BOOT_ONLY
+- chameleon::REQUEST_TYPE_FRONTEND
+- chameleon::REQUEST_TYPE_UNITTEST
+- ChameleonSystem\CoreBundle\RequestType\RequestTypeInterface::REQUEST_TYPE_BOOT_ONLY
+- ChameleonSystem\CoreBundle\RequestType\RequestTypeInterface::REQUEST_TYPE_UNITTEST
+- MTHeader::CONFIGPARAM_DB_COUNTER
+- MTHeader::CONFIGPARAM_TIMESTAMP
+- MTHeader::DB_LOGGING_STATE
+- MTHeader::TIMESTAMP_CREATED_IN_SESSION
+- PKG_EXTERNAL_TRACKER_GOOGLE_ANALYTICS_ENABLE_CROSS_DOMAIN_TRACKING
+- TCMSCronJob::LOG_FILE
+- TCMSMemcache::CACHE_DRIVER_MEMCACHE
+- TCMSMemcache::CACHE_DRIVER_MEMCACHED
+- TCMSUserInput::FILTER_EMAIL
+- TCMSUserInput::FILTER_XSS
+- TDataExtranetUser::MAX_SALT_AGE_IN_SECONDS
+
+## Classes and Interfaces
+
+- AbstractPkgCmsProfilerItem
+- CMSMediaViddlerImport
+- IClusterDriver
+- ICmsObjectLink
+- IMapperCacheManager
+- IMapperCacheManagerRestricted
+- IPkgCmsEventObservable
+- IPkgCmsServerSetupValidator
+- IPkgCmsServerSetupValidatorMessage
+- IPkgCoreDbClassFactory
+- IUserCustomModelBase
+- MapperCacheManager
+- MapperCacheManagerException
+- MapperCacheManagerExceptionContentNotFound
+- MapperCacheManagerRestrictedProxy
+- MTPkgExternalTrackerGoogleAnalytics_MTPageMetaCore
+- TCacheManagerStorage_Decorator
+- TCacheManagerStorage_Decorator_LazyWriteMemcache
+- TCacheManagerStorage_Standard
+- TCMSDataExtranetUser
+- TCMSMath
+- TCmsObjectLinkBase
+- TCmsObjectLinkException_InvalidTargetClass
+- TCMSSmartURLHandler_Pagepath
+- TCMSUserInput_EMail
+- TCMSUserInput_XSS
+- TPkgCmsClassManager_CmsConfig
+- TPkgCmsCore
+- TPkgCmsFileManagerException
+- TPkgCmsProfileItem_Group
+- TPkgCmsProfileItem_Tick
+- TPkgCmsProfiler
+- TPkgCmsServerSetupValidator_PHPVersion
+- TPkgCmsServerSetupValidatorManager
+- TPkgCmsServerSetupValidatorMessage
+- TPkgCmsSessionHandler_Decorator_Observable
+- TPkgExternalTrackerGoogleAnalytics
+
+## Properties
+
+- ChameleonSystem\CoreBundle\Controller\ChameleonController::$portalDomainService
+- ChameleonSystem\CoreBundle\Controller\ChameleonController::$redirectPageDef
+- ChameleonSystem\CoreBundle\Controller\ChameleonController::$requestInfoService
+- MTPageMetaCoreEndPoint::$oActivePortal
+- TCMSMemcache::$sUsedMemcacheClass
+- TGlobal::$oURLHistory
+- TGlobalBase::$aUnitTestMockedObjects
+- TModelBase::$isExportCall
+
+## Methods
+
+- ChameleonSystem\CoreBundle\Controller\ChameleonController::GetExecutionTime()
```

Comment:
The other PR has no () here...

Reply:
I prefer the variant with brackets, but we can still change near the end of the release.